board/common/uboot/scripts/ixbtn-factory.sh

```diff
@@ -0,0 +1,6 @@
+if button factory-reset; then
+    echo "Keep button pressed for 10 seconds to engage reset"
+    if sleep 10 && button factory-reset; then
+	run ixfactory
+    fi
+fi
```

Comment:
Customer requirement is 30 seconds, but maybe this is overridden in customer builds?

Also, maybe s/reset/factory reset/ to distinguish between reset/reboot command in U-Boot?

Reply:
Yes, these are broken out into separate scripts exactly to support easy customization - so no worries there!

But yes, that message should definitely be improved, thanks! :+1: